src/app/containers/BuyBTCPage/components/transak.ts

```diff
@@ -0,0 +1,32 @@
+import transakSDK from '@transak/transak-sdk';
+
+let transak = new transakSDK({
+  apiKey: process.env.TRANSAK_API_KEY, // Your API Key
+  environment: 'STAGING', // STAGING/PRODUCTION
+  defaultCryptoCurrency: 'ETH',
+  walletAddress: '', // Your customer's wallet address
+  themeColor: '000000', // App theme color
+  fiatCurrency: '', // INR/GBP
+  email: '', // Your customer's email address
+  redirectURL: '',
+  hostURL: window.location.origin,
+  widgetHeight: '550px',
+  widgetWidth: '450px',
+});
+
+export function openTransak() {
+  transak.init();
+
+  // To get all the events
+  transak.on(transak.ALL_EVENTS, data => {
+    console.log(data);
+  });
+
+  // This will trigger when the user marks payment is made.
+  transak.on(transak.EVENTS.TRANSAK_ORDER_SUCCESSFUL, orderData => {
+    console.log(orderData);
+    transak.close();
+  });
+
+  return true;
+}
```

Comment:
I get an error when writing `  return <></>;` Like it is not valid JS (it is HTML actually). Why don't you get the same error? :thinking:

Reply:
File extension then should be .tsx and function name start with uppercase - OpenTransak instead of openTransak.

To use this component use <OpenTransak /> instead of {openTransak()}.

Most likely useEffect didn't do its job because of this

Reply:
well, I guess then I need to put it back in the components folder